cmd/admin.go

```diff
@@ -366,6 +370,77 @@ var (
 			},
 		},
 	}
+
+	smtpCLIFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Value: "",
+			Usage: "Application Name",
+		},
+		cli.StringFlag{
+			Name:  "auth-type",
+			Value: "PLAIN",
+			Usage: "SMTP Authentication Type (PLAIN/LOGIN/CRAM-MD5) default PLAIN",
+		},
+		cli.StringFlag{
+			Name:  "host",
+			Value: "",
+			Usage: "SMTP Host",
+		},
+		cli.IntFlag{
+			Name:  "port",
+			Usage: "SMTP Port",
+		},
+		cli.StringFlag{
```

Comment:
Why don't you just use BoolFlag?

Reply:
> Why don't you just use BoolFlag?

I did try using BoolFlag but it does not accept --flag true/false so you have to use two separate flag for every bool eg. --flag --no-flag. Did it exactly as oauth setting but there is catch. 

1. If you always get value from BoolFlag you need to pass all options on update eg. you cant't update only one option. That's way runUpdateOauth is not using parseOAuth2Config. 
2. If you use IsSet() you can't disable BoolFlag option and that's way runUpdateOauth doesn't handle "skip-local-2fa".
3. Could not activate/deactivate it using runUpdateOauth

It would be great if BoolFlag could handle different cases:
1. --flag, --flag true; update to true
2. --flag false; update to false
3. No flag, no update needed

Reply:
There is BoolTFlag which is just what I need. Sorry for inconvenience.